include/vinecopulib/bicop/implementation/tawn.ipp

```diff
@@ -0,0 +1,71 @@
+// Copyright © 2016-2023 Thomas Nagler and Thibault Vatter
+//
+// This file is part of the vinecopulib library and licensed under the terms of
+// the MIT license. For a copy, see the LICENSE file in the root directory of
+// vinecopulib or https://vinecopulib.github.io/vinecopulib/.
+
+#include <vinecopulib/misc/tools_eigen.hpp>
+#include <vinecopulib/misc/tools_integration.hpp>
+
+namespace vinecopulib {
+inline TawnBicop::TawnBicop()
+{
+  family_ = BicopFamily::tawn;
+  parameters_ = Eigen::VectorXd(3);
+  parameters_lower_bounds_ = Eigen::VectorXd(3);
+  parameters_upper_bounds_ = Eigen::VectorXd(3);
+  parameters_ << 0.5, 0.5, 6;
```

Comment:
Those correspond to something specific?

Reply:
nah, should probably be (1, 1, 1) or so for independence (indep whenever parameters(2) = 1)